CMakeLists.txt

```
@@ -143,6 +102,44 @@ if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE RelWithDebInfo CACHE STRING "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel." FORCE)
 endif()
 
+##
+# Project version
+##
+set(COLOBOT_VERSION_CODENAME "Gold")
+set(COLOBOT_VERSION_RELEASE "alpha")
+
+if(COLOBOT_OFFICIAL_BUILD)
+    set(COLOBOT_VERSION_DISPLAY "${CMAKE_PROJECT_VERSION}-${COLOBOT_VERSION_RELEASE}")
+    set(COLOBOT_VERSION_FULL "${CMAKE_PROJECT_VERSION}-${COLOBOT_VERSION_RELEASE}")
+else()
+    # Append git characteristics to version string
+    if(EXISTS "${CMAKE_SOURCE_DIR}/.git")
+        find_package(Git)
+        execute_process(COMMAND ${GIT_EXECUTABLE} rev-parse --abbrev-ref HEAD
+                        WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
+                        OUTPUT_VARIABLE GIT_BRANCH
+                        OUTPUT_STRIP_TRAILING_WHITESPACE)
+        execute_process(COMMAND ${GIT_EXECUTABLE} rev-parse --short HEAD
+                        WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
+                        OUTPUT_VARIABLE GIT_REVISION
+                        OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+        set(COLOBOT_VERSION_DISPLAY "git-${GIT_BRANCH}~r${GIT_REVISION}")
+        set(COLOBOT_VERSION_FULL "${CMAKE_PROJECT_VERSION}-${COLOBOT_VERSION_RELEASE}-git-${GIT_BRANCH}~r${GIT_REVISION}")
+    else()
+        set(COLOBOT_VERSION_DISPLAY "${COLOBOT_VERSION_CODENAME}-${COLOBOT_VERSION_RELEASE}")
```

It looks like there used to be a + sign here

```suggestion
        set(COLOBOT_VERSION_DISPLAY "${COLOBOT_VERSION_CODENAME}-+${COLOBOT_VERSION_RELEASE}")
```